### PR TITLE
Ruby: update dependencies

### PR DESCRIPTION
**Pull request by aibaars**
Update the lock files to include the `codeql/ssa` library and replace ad-hoc pack create command with a proper `codeql pack install` command now the ssa library has been published.

### CODE REVIEW

File: .github/workflows/ruby-build.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ jobs:
         uses: ./.github/actions/fetch-codeql
       - name: Build Query Pack
         run: |
-          codeql pack create ../shared/ssa --output target/packs
+          codeql pack install ql/lib
```
Comment (hvitved) [inline .github/workflows/ruby-build.yml:98]: Will this still build the pack from the repo, or fetch it from the registry?

Comment (aibaars) [reply .github/workflows/ruby-build.yml:98]: I guess it fetches it from the registry.

Comment (hvitved) [reply .github/workflows/ruby-build.yml:98]: That doesn't work then. We do not guarantee, that the code in `main` is compatible with any released shared pack; only that it is compatible with the version of the pack at `main` as well.

Comment (aibaars) [reply .github/workflows/ruby-build.yml]: Ah ok. In that case we should remove the `codeql pack install ql/src` call too and use the suite-helpers from `../misc`

**Review by erik-krogh — DISMISSED** [review]
LGTM - except for the lock files.  
There is no need for explicit dependency versions in the lock files, as all the dependencies are present locally.  
So I removed those and pushed that to this PR.  

Ruby and Go were the only languages that had any explicit dependencies in their lockfiles, which I suppose are leftovers from the days of living in a separate repo.  

/cc @aeisenberg: Am I correct in assuming that these lock files don't need anything in `dependencies:`?

File: .github/workflows/ruby-build.yml [hunk 2]
```diff
@@ -202,13 +202,14 @@ jobs:
...
           dependencies:
-            codeql/ruby-all: 0.0.1
+            codeql/ruby-all: "*"
           extractor: ruby
           tests: .
           ' > qlpack.yml
       - name: Run QL test
         shell: bash
         run: |
+          codeql pack install .
```
Comment (aibaars) [inline .github/workflows/ruby-build.yml:212]: Let's see if we still need this:
```suggestion
```

**Review by erik-krogh — APPROVED** [review]

**Comment by aeisenberg** [comment]
> /cc @aeisenberg: Am I correct in assuming that these lock files don't need anything in `dependencies:`?

That's right!

**Pull request merged**